tests/models/deepseek_v3/test_modeling_deepseek_v3.py

```diff
@@ -605,6 +606,10 @@ def setUpClass(cls):
             # 8 is for A100 / A10 and 7 for T4
             cls.cuda_compute_capability_major_version = torch.cuda.get_device_capability()[0]
 
+    def tearDown(self):
+        # See LlamaIntegrationTest.tearDown(). Can be removed once LlamaIntegrationTest.tearDown() is removed.
+        cleanup(torch_device, gc_collect=False)
```

Comment:
For compilation, it seems to me better to also include

> torch._dynamo.reset()

see

https://github.com/pytorch/FBGEMM/pull/1992/files

If you agree, you can simply add a new key argument to `def cleanup`

Reply:
ah yes, good idea -- going to add to this PR!